tests/test_csp_ui_safety_check.py

```
@@ -0,0 +1,87 @@
+
+from __future__ import annotations
+
+import importlib.util
+import sys
+from pathlib import Path
+
+REPO = Path(__file__).resolve().parent.parent
+SCRIPT = REPO / "scripts" / "check_csp_ui_safety.py"
+
+
+def _load_module():
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_load_module`

(ANN202)